### PR TITLE
Usability improvements

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
- Edit agenda item number
- go to detail view for motion/assignment form
- replace all default checkboxes with fa-checkboxes
- goto line number now in popup
- added customization for category/motionBlock/tag in motion detail view.

### CODE REVIEW

File: openslides/assignments/static/templates/assignments/assignment-form.html [hunk 1]
```diff
@@ -5,8 +5,13 @@ <h1 ng-if="!model.id" translate>New election</h1>
   {{ alert.msg }}
 </div>
 
-<form name="assignmentForm" ng-submit="save(model)">
+<form name="assignmentForm" ng-submit="save(model, gotoDetailView)">
   <formly-form model="model" fields="formFields">
+    <hr class="smallhr">
+    <div class="checkbox pointer" ng-click="$parent.$parent.gotoDetailView = !$parent.$parent.gotoDetailView">
+      <i class="fa" ng-class="$parent.$parent.gotoDetailView ? 'fa-check-square-o' : 'fa-square-o'"></i>
+      <translate>Goto assignment detail view after save</translate>
```
Comment (emanuelschuetze) [inline openslides/assignments/static/templates/assignments/assignment-form.html]: Please use "Open election detail view after save."

File: openslides/motions/static/templates/motions/motion-form.html [hunk 2]
```diff
@@ -6,8 +6,13 @@ <h1 ng-if="parent"><translate>New amendment of motion</translate> {{ parent.iden
   {{ alert.msg }}
 </div>
 
-<form name="motionForm" ng-submit="save(model)" novalidate>
+<form name="motionForm" ng-submit="save(model, gotoDetailView)" novalidate>
   <formly-form model="model" fields="formFields">
+    <hr class="smallhr">
+    <div class="checkbox pointer" ng-click="$parent.$parent.gotoDetailView = !$parent.$parent.gotoDetailView">
+      <i class="fa" ng-class="$parent.$parent.gotoDetailView ? 'fa-check-square-o' : 'fa-square-o'"></i>
+      <translate>Goto motion detail view after save</translate>
```
Comment (emanuelschuetze) [inline openslides/motions/static/templates/motions/motion-form.html]: Please use "Open motion detail view after save."

File: openslides/assignments/static/templates/assignments/assignment-form.html (hunk 1 above)
Comment (FinnStutzenstein) [inline openslides/assignments/static/templates/assignments/assignment-form.html]: Done

File: openslides/motions/static/templates/motions/motion-form.html (hunk 2 above)
Comment (FinnStutzenstein) [inline openslides/motions/static/templates/motions/motion-form.html]: Done.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Done.

**Comment by normanjaeckel** [comment]
Please rebase. Then merge.

**Comment by emanuelschuetze** [comment]
Rebased.

**Pull request merged**